packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -199,6 +199,38 @@ class GuildBanManager extends CachedManager {
     await this.client.rest.delete(Routes.guildBan(this.guild.id, id), { reason });
     return this.client.users.resolve(user);
   }
+
+  /**
+   * Ban up to 200 users from a guild, and optionally delete previous messages sent by the banned users.
```

Comment:
We don’t usually write the limits of the API into the JSDoc and even less do we check the size of what we pass to the API before. The API will error itself anyway. But if discord changes those limits we‘d fail unnecessarily here and the docs would be wrong too.